sources/api/settings-committer/src/main.rs

```diff
@@ -191,7 +191,7 @@ fn parse_args(args: env::Args) -> Args {
 
     Args {
         transaction: transaction.unwrap_or_else(|| constants::LAUNCH_TRANSACTION.to_string()),
-        log_level: log_level.unwrap_or(LevelFilter::Info),
+        log_level: log_level.unwrap_or(LevelFilter::Trace),
```

Comment:
Looks like a debugging/testing change to be undone.

Reply:
yup good catch